tests/test_tranform.py

```diff
@@ -32,29 +34,8 @@ def scan_html_datadir():
 html_dir_data = scan_html_datadir()
 
 
-def test_to_html():
-    """Test to check to_html() function"""
-    input_data = json.dumps(json_dir_data['simple'])
-    expected_html = html_dir_data['simple']
-    assert tiptapy.to_html(input_data) == expected_html
-
-
-def test_blockquote():
-    """Test to check BlockQuote"""
-    blockquote = json.dumps(json_dir_data['blockquote'])
-    expected_html = html_dir_data['blockquote']
-    assert tiptapy.to_html(blockquote) == expected_html
-
-
-def test_bulletlist():
-    """Test to check BulletList"""
-    bulletlist = json.dumps(json_dir_data['bulletlist'])
-    expected_html = html_dir_data['bulletlist']
-    assert tiptapy.to_html(bulletlist) == expected_html
-
-
-def test_text():
-    """Test to check mark_tags bold,italic,link"""
-    mark_tags = json.dumps(json_dir_data['mark_tags'])
-    expected_html = html_dir_data['mark_tags']
-    assert tiptapy.to_html(mark_tags) == expected_html
+@pytest.mark.parametrize("tag", ["simple", "blockquote", "bulletlist", "mark_tags"])
```

Comment:
Parametrization 👍 . 
One more suggestion, rename html_dir_data to html_data. Same for json_dir_data

Reply:
updated!